src/community/label_propagation.c

```diff
@@ -337,6 +347,8 @@ igraph_error_t igraph_community_label_propagation(const igraph_t *graph,
             }
         }
         RNG_END();
+
+        control_iteration = !control_iteration;
```

Comment:
Could you also please put a comment here to say we are alternating between control iterations and label updating iterations?